client/src/Clusters/AppOfTheWeek/AOTWStyle.css

```diff
@@ -0,0 +1,21 @@
+.aotw{
+    background-image: 
+        linear-gradient(to right, #C4E9DE 1%, #F0F9F700), 
+            url(./Images/background-image.jpg);
+    text-align: left;
+    margin: 100px;
```

Comment:
Why margin 100px? Seems a bit arbitrary

Reply:
It is, i kinda needed it to look like the mock